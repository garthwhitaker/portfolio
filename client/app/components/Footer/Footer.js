import React from "react";
import Grid from "@material-ui/core/Grid";
import withStyles from "@material-ui/styles/withStyles";
import PropTypes from "prop-types";

const styles = {
  row: {
    display: "flex",
    justifyContent: "center",
    fontFamily: '"Open Sans", sans-serif',
  },
  divider: {
    marginTop: "1rem",
    marginBottom: "1rem",
    border: 0,
    borderTop: "1px solid rgba(0,0,0,.1)",
  },
};

const Footer = ({ classes: { divider, row } }) => (
  <div>
    <hr className={divider} />
    <Grid container>
      <Grid className={row} item xs={12}>
        &copy; Copyright 2020 Garth Whitaker
      </Grid>
    </Grid>
  </div>
);

Footer.propTypes = {
  classes: PropTypes.shape({
    row: PropTypes.string.isRequired,
    divider: PropTypes.string.isRequired,
  }).isRequired,
};

export default withStyles(styles)(Footer);
