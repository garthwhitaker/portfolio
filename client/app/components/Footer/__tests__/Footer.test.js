import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './../Footer';

describe('Footer', () => {
  test('it renders correctly', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
