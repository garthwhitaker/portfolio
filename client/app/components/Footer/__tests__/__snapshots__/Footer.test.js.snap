// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer it renders correctly 1`] = `
<div>
  <hr
    className="Footer-divider-2"
  />
  <div
    className="MuiGrid-root MuiGrid-container"
  >
    <div
      className="MuiGrid-root Footer-row-1 MuiGrid-item MuiGrid-grid-xs-12"
    >
      Â© Copyright 2018 Garth Whitaker
    </div>
  </div>
</div>
`;
